src/Compilers/Core/Portable/RuleSet/RuleSetInclude.cs

```diff
@@ -80,6 +80,11 @@ private string GetIncludePath(RuleSet parent)
             List<string> found = new List<string>();
             string expandedPath = PortableShim.Environment.ExpandEnvironmentVariables(_includePath);
 
+            if (PathUtilities.IsUnixLikePlatform)
+            {
+                expandedPath = expandedPath.Replace('\\', PortableShim.Path.DirectorySeparatorChar);
```

Comment:
We want existing ruleset files to work cross platform - so that users can open existing projects in Xamarin Studio on Mac, for example. So we need to handle both separators.

How about we first check if the file exists as specified. If it doesn't let's make the ```\``` to ```/``` replacement in the original string and repeat the process. This won't handle crazy corner cases like mixed separators in Linux-mounted Windows partitions, but I think that's fine.

BTW, I noticed we call ```ExpandEnvironmentVariables```. I bet Linux implementation of this method doesn't handle %VAR% syntax, or does it? I guess we can hope that not many customers actually use this feature and leave it Windows specific for now. Perhaps file an issue to track this as a potential cross platform problem.

I also noticed that this class is public yet doesn't any argument validation. Also Path APIs throw on invalid characters, we should handle that and report diagnostics rather than crash.